packages/core-ethereum/src/ethereum.ts

```diff
@@ -133,6 +134,7 @@ export async function createChainWrapper(
     }
     log('essentialTxPayload %o', essentialTxPayload)
 
+    let removeListener
```

Comment:
Please make sure that `removeListener` is always defined.

Reply:
Close this discussion due to changes in the implementation